pact/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -2022,10 +2026,15 @@ isKeysetInSigs info cont handler env (KeySet kskeys ksPred) = do
   where
   matchKey k _ = k `elem` kskeys
   atLeast t m = m >= t
+  elide pk | T.length pk < 8 = pk
+           | otherwise = T.take 8 pk <> "..."
   count = S.size kskeys
   run p matched =
     if p count matched then returnCEKValue cont handler (VBool True)
-    else returnCEK cont handler (VError "Keyset enforce failure" info)
+    else let
+      errMsg = "Keyset failure (" <> predicateToText ksPred <> "): "
```

Comment:
I like this but it would break our tests:
```
   marmalade-runner.repl:                                                   FAIL (0.04s)
        pact-tests/Pact/Core/Test/ReplTests.hs:111:
        marmalade-runner.repl:279:2: Program encountered an unhandled raised error: FAILURE: offer fails if OFFER capability is not signed by seller: expected error message 'Keyset failure (keys-all): [e4c6807d...]', got 'Keyset failure'        
```
as the marmalade expectations are very precise, see here: https://github.com/kadena-io/marmalade/blob/3eaaf08cd71304f43bc87ff97a0b29f776ef253b/pact/marmalade.repl#L280

My proposal would be we keep it as is for the time being and address this in a subsequent PR. 
What do you think?